test/e2e/storage/testsuites/multivolume.go

```diff
@@ -107,6 +107,10 @@ func (t *multiVolumeTestSuite) DefineTests(driver TestDriver, pattern testpatter
 		// Now do the more expensive test initialization.
 		l.config, l.driverCleanup = driver.PrepareTest(f)
 		l.intreeOps, l.migratedOps = getMigrationVolumeOpCounts(f.ClientSet, dInfo.InTreePluginName)
+		keys := dInfo.TopologyKeys
+		if len(keys) == 0 {
+			framework.Skipf("Driver %s didn't provide topology keys -- skipping", dInfo.Name)
+		}
```

Comment:
Can we assume that volumes of drivers without topology keys can run on any node? We need to fix all the "cloud" ones to include topology in test/e2e/storage/drivers/in_tree.go, only GCE sets the correct keys:

https://github.com/kubernetes/kubernetes/blob/6a040438ea8837c5990f6b3e122ebe349fbcf409/test/e2e/storage/drivers/in_tree.go#L1161-L1171

Reply:
Fixed